src/Test.UnitTests.Sarif/FileRegionsCacheTests.cs

```diff
@@ -311,15 +312,49 @@ private static void ExecuteTests(ReadOnlyCollection<TestCaseData>  testCases, Fi
         }
 
         [Fact]
-        public void FileRegionsCache_PopulatesNullRegion()
+        public void FileRegionsCache_ProperlyCaches()
         {
+            Uri uri = new Uri(@"C:\Code\Program.cs");
+
+            StringBuilder fileContents = new StringBuilder();
+            for (int i = 0; i < 1000; ++i)
+            {
+                fileContents.AppendLine("0123456789");
+            }
+
             var run = new Run();
-            var fileRegionsCache = new FileRegionsCache(run);
+            var mockFileSystem = MockFactory.MakeMockFileSystem(uri.LocalPath, fileContents.ToString());
```

Comment:
>var [](start = 12, length = 3)

How about replace with type here?

Reply:
Sure. I had moved this code up and left it alone, but happy to provide the resolved types.